.travis.yml

```diff
@@ -8,18 +8,18 @@ env:
 matrix:
   include:
     - env: ENV=vim74
-      rvm: 1.8.7
+      rvm: 2.1.5
```

Comment:
1.8.7 didn't work in Xenial. Is there a usecase to continue testing ruby versions before 2.1?

Reply:
Probably not, but how about moving `dist` into `matrix` so that both distro versions are tested with their accompanying Ruby versions?


```yaml
    - env: ENV=vim74
      dist: trusty
      rvm: 1.8.7
      addons: { apt: { packages: [vim-nox] } }
    - env: ENV=vim74
      dist: xenial
      rvm: 2.1.5
      addons: { apt: { packages: [vim-nox] } }
...
```

Ref: https://docs.travis-ci.com/user/multi-os/